### PR TITLE
*sparkles* feat: add verification for fulcio issued cert

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

* Adds verification for fulcio issued certificates

### CODE REVIEW

File: app/post_results.go [hunk 1]
```diff
@@ -415,3 +437,18 @@ func extractCerts(entry *tlogEntry) ([]*x509.Certificate, error) {
 	}
 	return result, nil
 }
+
+func getFulcioRoots() (*x509.CertPool, *x509.CertPool, error) {
```
Comment (azeemshaikh38) [inline app/post_results.go:441]: For better code readability, could you instead make the fn signature to be:

```
func getCertPool([]byte) (*x509.CertPool, error) {}
```

**Review by azeemshaikh38 — APPROVED** [review]
Thanks, LGTM. Probably need to run `go mod tidy && go mod verify` to fix the CI failures.

**Comment by asraa** [comment]
> Probably need to run go mod tidy && go mod verify to fix the CI failures.

interesting, ran this both with my local go and go1.17.7 as used in CI and no change occurred. i'll look deeper

File: app/post_results.go [hunk 2]
```diff
@@ -74,6 +74,12 @@ type tlogEntry struct {
 	IntegratedTime int64  `json:"integratedTime"`
 }
 
+//go:embed fulcio_v1.crt.pem
+var fulcioRoot []byte
+
+//go:embed fulcio__intermediate.crt.pem
```
Comment (azeemshaikh38) [inline app/post_results.go:80]: Typo - double underscore ?

Comment (asraa) [reply app/post_results.go]: interesting... i think that fixed the typecheck too!

**Pull request merged**